ui/apps/platform/src/Containers/NetworkGraph/TopologyComponent.tsx

```diff
@@ -126,6 +125,10 @@ const TopologyComponent = ({
         onSelect(ids);
     });
 
+    useEffect(() => {
+        controller.fromModel(model);
+    }, [controller, model]);
+
```

Comment:
this was to prevent a console error ("Cannot update a component (`StyleNode`) while rendering a different component (`TopologyComponent`). To locate the bad setState() call inside `TopologyComponent`, follow the stack trace as described in https://reactjs.org/link/setstate-in-render")